src/main/resources/openapi.json

```diff
@@ -1462,6 +1499,61 @@
                         "topic2"
                     ]
                 }
+            },
+            "SubscribedTopic": {
+                "title": "SubscribedTopic",
+                "type": "object",
+                "properties": {
+                    "my-topic1": {
+                        "type": "array",
+                        "items": {
+                            "format": "int32",
+                            "type": "integer"
+                        }
+                    }
+                },
+                "example": {
+                    "my-topic1": [
+                        1,
+                        2,
+                        3
+                    ]
+                }
+            },
+            "SubscribedTopicList": {
+                "title": "SubscribedTopicList",
+                "type": "object",
+                "properties": {
+                    "topics": {
+                        "$ref": "#/components/schemas/Topics"
```

Comment:
Where is this Topics declaration. Anyway I don't think we need it, it's just an array of strings. How we do for subscription request for example?

Reply:
We already had the Topics data type there (for subscription of topic list).